cosmos/ingestion/ingest/ingest.py

```diff
@@ -206,9 +210,11 @@ def raise_timeout(var1, var2):
                         dataset_id=dataset_id,
                         pp_threshold=pp_threshold,
                         d_threshold=d_threshold,
-                        spans=spans)
+                        spans=spans,
+                        qa=qa)
 
-    def enrich(self, file_path: str, dataset_id: str, pp_threshold: float, d_threshold: float, spans: int):
+    def enrich(self, file_path: str, dataset_id: str, pp_threshold: float, d_threshold: float,
```

Comment:
Can we move this (and any other enrichment-centered pieces) into their own file, similar to `aggregate_router`, `detect`, etc.?

Reply:
broke out enrichment code to ingest.process.enrich.context_enrichment

Reply:
@iross should be set on this one.